.readthedocs.yaml

```diff
@@ -0,0 +1,16 @@
+version: 2
+
+# Build from the docs/ directory with Sphinx
+sphinx:
+    configuration: docs/source/conf.py
+
+build:
+    image: latest
+    apt_packages:
+        - doxygen
+
+# Explicitly set the version of Python and its requirements
+python:
+    version: 3.7
```

Comment:
Do we want to explicitly set python version?  On my machine I'm running 3.8 and I'm not sure if that would cause issues.

Reply:
This is purely for read the docs which is the service that hosts the website, so no one will be impacted by the python version.